[Bug]: Installation issue with "Installing with a conda environment.yml file" option

### FAQ check

- [X] Yes, I checked the FAQ and my question has not been addressed.

### Instrument

Other (any stage)

### What happened?

Kevin Stevenson and I discovered a bug while installing Eureka! using the "Installing with a conda environment.yml file" option on the Eureka! Installation page. Using this method, Eureka! was successfully installed, but when I tried to test the installation by running "pytest test_NIRCam.py", I received two errors stating that there were no modules named 1) "eureka.version" and 2) "setuptools_scm". We realized these packages were not on the "environment.yml" file and do not get installed using this option. However, they are installed using method A (git and pip) on the installation page. After running "pip install -e '.[jwst]'", this error was corrected. I no longer have an issue now that we have solved this puzzle, but this should be addressed on the "Installing with a conda environment.yml file" option. Thank you! 

### Error traceback output

../src/eureka/__init__.py:5: in <module>
    from .version import __version__
E   ModuleNotFoundError: No module named 'eureka.version'

During handling of the above exception, another exception occurred:
../../../../../opt/anaconda3/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_NIRCam.py:9: in <module>
    from eureka.lib.readECF import MetaClass
../src/eureka/__init__.py:7: in <module>
    from setuptools_scm import get_version
E   ModuleNotFoundError: No module named 'setuptools_scm'

### What operating system are you using?

Mac OS 12.5

### What version of Python are you running?

Python 3.9.7

### What Python packages do you have installed?

# Name                    Version                   Build  Channel
_ipython_minor_entry_point 8.7.0                h8cf3c4a_0    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
aom                       3.5.0                hf0c8a7f_0    conda-forge
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
arviz                     0.14.0             pyhd8ed1ab_0    conda-forge
asdf                      2.14.1             pyhd8ed1ab_0    conda-forge
asdf-standard             1.0.3              pyhd8ed1ab_0    conda-forge
asdf-transform-schemas    0.3.0              pyhd8ed1ab_0    conda-forge
asdf-unit-schemas         0.1.0              pyhd8ed1ab_0    conda-forge
asteval                   0.9.27             pyhd8ed1ab_0    conda-forge
astropy                   5.1.1            py39h7cc1f47_3    conda-forge
astropy-healpix           0.7              py39h7cc1f47_2    conda-forge
astroscrappy              1.1.0            py39h7cc1f47_2    conda-forge
asttokens                 2.2.0              pyhd8ed1ab_0    conda-forge
attrs                     22.1.0             pyh71513ae_1    conda-forge
babel                     2.11.0             pyhd8ed1ab_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                pyhd8ed1ab_3    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
backports.zoneinfo        0.2.1            py39h6e9494a_7    conda-forge
batman-package            2.4.9            py39h7cc1f47_1    conda-forge
beautifulsoup4            4.11.1             pyha770c72_0    conda-forge
bleach                    5.0.1              pyhd8ed1ab_0    conda-forge
blosc                     1.21.1               h97e831e_3    conda-forge
bokeh                     2.4.3              pyhd8ed1ab_3    conda-forge
bottleneck                1.3.5            py39h7cc1f47_1    conda-forge
brotli                    1.0.9                hb7f2c08_8    conda-forge
brotli-bin                1.0.9                hb7f2c08_8    conda-forge
brotlipy                  0.7.0           py39ha30fb19_1005    conda-forge
brunsli                   0.1                  h046ec9c_0    conda-forge
bzip2                     1.0.8                h0d85af4_4    conda-forge
c-ares                    1.18.1               h0d85af4_0    conda-forge
c-blosc2                  2.5.0                had5f71b_0    conda-forge
ca-certificates           2022.9.24            h033912b_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
ccdproc                   2.4.0              pyhd8ed1ab_0    conda-forge
celerite                  0.4.2            py39hecff1ad_3    conda-forge
certifi                   2022.9.24          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1           py39h131948b_2    conda-forge
cfitsio                   4.2.0                hd56cc12_0    conda-forge
cftime                    1.6.2            py39h7cc1f47_1    conda-forge
charls                    2.3.4                he49afe7_0    conda-forge
charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
click                     8.1.3           unix_pyhd8ed1ab_2    conda-forge
cloudpickle               2.2.0              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
contourpy                 1.0.6            py39h92daf61_0    conda-forge
corner                    2.2.1              pyhd8ed1ab_0    conda-forge
coverage                  6.5.0            py39ha30fb19_1    conda-forge
cryptography              38.0.3           py39h7eb6a14_0    conda-forge
curl                      7.86.0               h57eb407_1    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
cython                    0.29.32          py39h7a8716b_1    conda-forge
cytoolz                   0.12.0           py39ha30fb19_1    conda-forge
dask-core                 2022.12.0          pyhd8ed1ab_0    conda-forge
dav1d                     1.0.0                hac89ed1_1    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
docutils                  0.19             py39h6e9494a_1    conda-forge
dynesty                   2.0.3              pyhd8ed1ab_0    conda-forge
emcee                     3.1.3              pyhd8ed1ab_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
exceptiongroup            1.0.4              pyhd8ed1ab_0    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
fonttools                 4.38.0           py39ha30fb19_1    conda-forge
freetype                  2.12.1               h3f81eb7_1    conda-forge
fsspec                    2022.11.0          pyhd8ed1ab_0    conda-forge
future                    0.18.2             pyhd8ed1ab_6    conda-forge
george                    0.4.0            py39hecff1ad_5    conda-forge
giflib                    5.2.1                hbcb3906_2    conda-forge
gwcs                      0.17.1             pyhd8ed1ab_0    conda-forge
h5py                      3.1.0           nompi_py39hdc2b67d_100    conda-forge
hdf4                      4.2.15               h7aa5921_5    conda-forge
hdf5                      1.10.6          nompi_hc5d9132_1114    conda-forge
hypothesis                6.60.0             pyha770c72_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
imagecodecs               2022.9.26        py39h8036967_4    conda-forge
imageio                   2.22.4             pyhfa7a67d_1    conda-forge
imagesize                 1.4.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        5.1.0              pyha770c72_0    conda-forge
importlib-resources       5.10.0             pyhd8ed1ab_0    conda-forge
importlib_resources       5.10.0             pyhd8ed1ab_0    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
ipython                   8.7.0              pyhd1c38e8_0    conda-forge
jedi                      0.18.2             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
jmespath                  1.0.1              pyhd8ed1ab_0    conda-forge
joblib                    1.2.0              pyhd8ed1ab_0    conda-forge
jpeg                      9e                   hac89ed1_2    conda-forge
jsonschema                4.17.3             pyhd8ed1ab_0    conda-forge
jupyter_client            7.4.8              pyhd8ed1ab_0    conda-forge
jupyter_core              5.1.0            py39h6e9494a_0    conda-forge
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jxrlib                    1.1                  h35c211d_2    conda-forge
kiwisolver                1.4.4            py39h92daf61_1    conda-forge
krb5                      1.19.3               hb49756b_0    conda-forge
lcms2                     2.14                 h90f4b2a_0    conda-forge
lerc                      4.0.0                hb486fe8_0    conda-forge
libaec                    1.0.6                he49afe7_0    conda-forge
libavif                   0.11.1               he9a59b1_0    conda-forge
libblas                   3.9.0           16_osx64_openblas    conda-forge
libbrotlicommon           1.0.9                hb7f2c08_8    conda-forge
libbrotlidec              1.0.9                hb7f2c08_8    conda-forge
libbrotlienc              1.0.9                hb7f2c08_8    conda-forge
libcblas                  3.9.0           16_osx64_openblas    conda-forge
libcurl                   7.86.0               h57eb407_1    conda-forge
libcxx                    14.0.6               hccf4f1f_0    conda-forge
libdeflate                1.14                 hb7f2c08_0    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.4.2                h0d85af4_5    conda-forge
libgfortran               5.0.0           9_5_0_h97931a8_26    conda-forge
libgfortran5              11.3.0              h082f757_26    conda-forge
liblapack                 3.9.0           16_osx64_openblas    conda-forge
libnetcdf                 4.8.1           nompi_hb4d10b0_100    conda-forge
libnghttp2                1.47.0               h7cbc4dc_1    conda-forge
libopenblas               0.3.21          openmp_h429af6e_3    conda-forge
libpng                    1.6.39               ha978bb4_0    conda-forge
libsodium                 1.0.18               hbcb3906_1    conda-forge
libsqlite                 3.40.0               ha978bb4_0    conda-forge
libssh2                   1.10.0               h7535e13_3    conda-forge
libtiff                   4.4.0                hdb44e8a_4    conda-forge
libwebp-base              1.2.4                h775f41a_0    conda-forge
libxcb                    1.13              h0d85af4_1004    conda-forge
libzip                    1.9.2                h3ad4413_1    conda-forge
libzlib                   1.2.13               hfd90126_4    conda-forge
libzopfli                 1.0.3                h046ec9c_0    conda-forge
llvm-openmp               15.0.6               h61d9ccf_0    conda-forge
lmfit                     1.1.0              pyhd8ed1ab_0    conda-forge
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
lz4-c                     1.9.3                he49afe7_1    conda-forge
markupsafe                2.1.1            py39ha30fb19_2    conda-forge
matplotlib                3.6.2            py39h6e9494a_0    conda-forge
matplotlib-base           3.6.2            py39hb2f573b_0    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mistune                   2.0.4              pyhd8ed1ab_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
nbclient                  0.7.2              pyhd8ed1ab_0    conda-forge
nbconvert                 7.2.6              pyhd8ed1ab_0    conda-forge
nbconvert-core            7.2.6              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.2.6              pyhd8ed1ab_0    conda-forge
nbformat                  5.7.0              pyhd8ed1ab_0    conda-forge
nbsphinx                  0.8.10             pyhd8ed1ab_0    conda-forge
ncurses                   6.3                  h96cf925_1    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
netcdf4                   1.5.7           nompi_py39h8cf34ae_101    conda-forge
networkx                  2.8.8              pyhd8ed1ab_0    conda-forge
numpy                     1.23.5           py39hdfa1d0c_0    conda-forge
openjpeg                  2.5.0                h5d0d7b0_1    conda-forge
openssl                   1.1.1s               hfd90126_1    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.5.2            py39hecff1ad_0    conda-forge
pandoc                    2.19.2               h694c41f_1    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
partd                     1.3.0              pyhd8ed1ab_0    conda-forge
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
photutils                 1.5.0            py39h15b18c7_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.2.0            py39h35d4919_3    conda-forge
pip                       22.3.1             pyhd8ed1ab_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_0    conda-forge
platformdirs              2.5.2              pyhd8ed1ab_1    conda-forge
pluggy                    1.0.0              pyhd8ed1ab_5    conda-forge
prompt-toolkit            3.0.33             pyha770c72_0    conda-forge
psutil                    5.9.4            py39ha30fb19_0    conda-forge
pthread-stubs             0.4               hc929b4f_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pyerfa                    2.0.0.1          py39h7cc1f47_3    conda-forge
pygments                  2.13.0             pyhd8ed1ab_0    conda-forge
pyopenssl                 22.1.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pyrsistent                0.19.2           py39ha30fb19_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
pytest                    7.2.0              pyhd8ed1ab_2    conda-forge
pytest-arraydiff          0.3                        py_0    conda-forge
pytest-astropy            0.10.0             pyhd8ed1ab_0    conda-forge
pytest-astropy-header     0.1.2                      py_0    conda-forge
pytest-cov                4.0.0              pyhd8ed1ab_0    conda-forge
pytest-doctestplus        0.12.1             pyhd8ed1ab_0    conda-forge
pytest-filter-subpackage  0.1.1                      py_0    conda-forge
pytest-mock               3.10.0             pyhd8ed1ab_0    conda-forge
pytest-openfiles          0.5.0                      py_0    conda-forge
pytest-remotedata         0.3.3              pyhd8ed1ab_0    conda-forge
python                    3.9.7           h1248fe1_3_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.16.2             pyhd8ed1ab_0    conda-forge
python_abi                3.9                      3_cp39    conda-forge
pytz                      2022.6             pyhd8ed1ab_0    conda-forge
pywavelets                1.3.0            py39h7cc1f47_2    conda-forge
pyyaml                    6.0              py39ha30fb19_5    conda-forge
pyzmq                     24.0.1           py39hed8f129_1    conda-forge
readline                  8.1.2                h3899abd_0    conda-forge
reproject                 0.9.1            py39h7cc1f47_2    conda-forge
requests                  2.28.1             pyhd8ed1ab_1    conda-forge
scikit-image              0.19.3           py39hecff1ad_2    conda-forge
scikit-learn              1.1.3            py39hdbdcc14_1    conda-forge
scipy                     1.9.3            py39h8a15683_2    conda-forge
semantic_version          2.10.0             pyhd8ed1ab_0    conda-forge
setuptools                65.5.1             pyhd8ed1ab_0    conda-forge
setuptools-scm            7.0.5              pyhd8ed1ab_1    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snappy                    1.1.9                h225ccf5_2    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
sphinx                    5.3.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_2    conda-forge
sqlite                    3.40.0               h9ae0607_0    conda-forge
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tifffile                  2022.10.10         pyhd8ed1ab_0    conda-forge
tinycss2                  1.2.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h5dbffcc_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
tornado                   6.2              py39ha30fb19_1    conda-forge
tqdm                      4.64.1             pyhd8ed1ab_0    conda-forge
traitlets                 5.6.0              pyhd8ed1ab_0    conda-forge
typing-extensions         4.4.0                hd8ed1ab_0    conda-forge
typing_extensions         4.4.0              pyha770c72_0    conda-forge
tzdata                    2022g                h191b570_0    conda-forge
uncertainties             3.1.7              pyhd8ed1ab_0    conda-forge
unicodedata2              15.0.0           py39ha30fb19_0    conda-forge
urllib3                   1.26.13            pyhd8ed1ab_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.38.4             pyhd8ed1ab_0    conda-forge
xarray                    2022.12.0          pyhd8ed1ab_0    conda-forge
xarray-einstats           0.3.0              pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.9                h35c211d_0    conda-forge
xorg-libxdmcp             1.1.3                h35c211d_0    conda-forge
xz                        5.2.6                h775f41a_0    conda-forge
yaml                      0.2.5                h0d85af4_2    conda-forge
zeromq                    4.3.4                he49afe7_1    conda-forge
zfp                       1.0.0                ha894c9a_3    conda-forge
zipp                      3.11.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               hfd90126_4    conda-forge
zlib-ng                   2.0.6                hac89ed1_0    conda-forge
zstd                      1.5.2                hfa58983_4    conda-forge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/docs/source/installation.rst
+++ b/docs/source/installation.rst
@@ -6,8 +6,8 @@ Installation methods
 --------------------
 
 In order to have consistent, repeatable results across the ``Eureka!`` user community, we recommend that all general users install
-the most recent stable release of ``Eureka!``, v0.5. The following installation instructions are written with this in mind,
-and the most recent stable release is also available as a zipped archive `here <https://github.com/kevin218/Eureka/releases/tag/v0.5>`_.
+the most recent stable release of ``Eureka!``, v0.7. The following installation instructions are written with this in mind,
+and the most recent stable release is also available as a zipped archive `here <https://github.com/kevin218/Eureka/releases/tag/v0.7>`_.
 Also note that if you are using a macOS device with an M1 processor, you will need to use the ``conda`` environment.yml file
 installation instructions below as the pip dependencies fail to build on the M1 processor.
 
@@ -31,7 +31,7 @@ Once in your new conda environment, you can install ``Eureka!`` directly from so
 
 .. code-block:: bash
 
-	git clone -b v0.5 https://github.com/kevin218/Eureka.git
+	git clone -b v0.7 https://github.com/kevin218/Eureka.git
 	cd Eureka
 	pip install -e '.[jwst]'
 
@@ -49,7 +49,7 @@ Once in your new conda environment, you can install the ``Eureka!`` package with
 
 .. code-block:: bash
 
-	pip install -e git+https://github.com/kevin218/Eureka.git@v0.5#egg=eureka[jwst]
+	pip install -e git+https://github.com/kevin218/Eureka.git@v0.7#egg=eureka[jwst]
 
 Other specific branches can be installed using:
 
@@ -68,7 +68,7 @@ you'd prefer not to use ``pip`` to install dependencies. This can be done follow
 
 .. code-block:: bash
 
-	git clone -b v0.5 https://github.com/kevin218/Eureka.git
+	git clone -b v0.7 https://github.com/kevin218/Eureka.git
 	cd Eureka
 	conda env create --file environment.yml --force
 	conda activate eureka

--- a/environment.yml
+++ b/environment.yml
@@ -36,6 +36,7 @@ dependencies:
         - image_registration @ git+https://github.com/keflavich/image_registration.git@master#egg=image_registration
         - jwst==1.6.0
         - myst-parser # Needed for documentation
+        - setuptools_scm # Needed for version number
         - sphinx-rtd-theme # Needed for documentation
         - stcal
         - stdatamodels

